extension/README.md

```diff
@@ -225,49 +225,32 @@ When this command is  executed, VSCode...
 
 The same feature as above, but with debugging
 
-#### NAB: Edit Xliff Document (preview feature)
+#### NAB: Edit Xliff Document
 
 Opens XLF-files for editing in a webview.
 
 With the goal of reducing the clutter of XML files this feature is built for translators or non-developers in mind. Command available from right clicking a XLF-file and command palette. 
-
+![Edit Xliff Document](images/gifs/XliffEditorUsage.gif)
```

Comment:
Checkout https://github.com/jwikman/nab-al-tools/blob/1d326167d2e349453d5be8e7f4f62b2ad4340799/extension/images/gifs/README.md ;-)

I couldn't see the full image on my laptop :)